buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -90,6 +91,13 @@ abstract class AndroidTarget extends JavaLibTarget {
         isKotlin = project.plugins.hasPlugin(KotlinAndroidPluginWrapper.class)
         hasKotlinAndroidExtensions = project.plugins.hasPlugin(KOTLIN_ANDROID_EXTENSIONS_MODULE)
 
+        try {
+            hasExperimentalKotlinAndroidExtensions = hasKotlinAndroidExtensions &&
+                                                     project.androidExtensions.experimental
+        } catch (Exception ignored) {
```

Comment:
Do you still need the catch block?

Reply:
We can probably get rid of the try-catch all together but I'm not sure if there are cases that I'm not testing for where `hasKotlinAndroidExtension` is true but `project.androidExtensions` does not exist